src/pages/AddItem.js

```diff
@@ -89,15 +91,91 @@ export default function AddItem({ token }) {
+          When will you purchase it?
+        </h2>
+
+        <div className="flex items-center w-full mb-5">
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            className="h-10 w-10 mr-5 fill-current text-caribbean-green"
+            viewBox="0 0 20 20"
+          >
+            <path
+              fillRule="evenodd"
+              d="M10 18a8 8 0 100-16 8 8 0 000 16zm1-12a1 1 0 10-2 0v4a1 1 0 00.293.707l2.828 2.829a1 1 0 101.415-1.415L11 9.586V6z"
+              clipRule="evenodd"
+            />
+          </svg>
+          <div className="flex items-center p-2 bg-gray-200 rounded w-full">
+            <input
+              type="radio"
+              name="frequency"
+              id="soon"
+              value={7}
+              // defaultChecked
+              className="h-5 w-5 mx-5"
+              onChange={handleFrequencyChange}
+            />
+            <label htmlFor="soon" className="text-midnight-green text-xl">
+              Soon
+            </label>
+          </div>
+        </div>
+
+        <div className="flex items-center w-full mb-5">
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            className="h-10 w-10 mr-5 fill-current text-orange-yellow"
+            viewBox="0 0 20 20"
+          >
+            <path
+              fillRule="evenodd"
+              d="M10 18a8 8 0 100-16 8 8 0 000 16zm1-12a1 1 0 10-2 0v4a1 1 0 00.293.707l2.828 2.829a1 1 0 101.415-1.415L11 9.586V6z"
+              clipRule="evenodd"
+            />
+          </svg>
+          <div className="flex items-center p-2 bg-gray-200 rounded w-full">
+            <input
+              type="radio"
+              name="frequency"
+              id="soonish"
+              value={14}
+              className="h-5 w-5 mx-5"
+              onChange={handleFrequencyChange}
+            />
+            <label htmlFor="soonish" className="text-midnight-green text-xl">
+              Soonish
+            </label>
+          </div>
+        </div>
+
+        <div className="flex items-center w-full mb-10">
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            className="h-10 w-10 mr-5 fill-current text-paradise-pink"
+            viewBox="0 0 20 20"
+          >
+            <path
+              fillRule="evenodd"
+              d="M10 18a8 8 0 100-16 8 8 0 000 16zm1-12a1 1 0 10-2 0v4a1 1 0 00.293.707l2.828 2.829a1 1 0 101.415-1.415L11 9.586V6z"
+              clipRule="evenodd"
+            />
+          </svg>
+          <div className="flex items-center p-2 bg-gray-200 rounded w-full">
+            <input
+              type="radio"
+              name="frequency"
+              id="not soon"
+              value={30}
+              className="h-5 w-5 mx-5"
+              onChange={handleFrequencyChange}
+            />
+            <label htmlFor="not soon" className="text-midnight-green text-xl">
+              Not soon
+            </label>
+          </div>
+        </div>
```

Comment:
Everything is working great!

I have noticed that some people have pulled their Tailwind styling out into separate components and so when I add the rest of the icons, those will be in their own components, but for functionality right now, it seems like things are good here.